setup.py

```diff
@@ -4,9 +4,15 @@
 import re
 
 
+# def local_file(filename):
```

Comment:
just remove the `local_file` function altogether do the version code in a `with open` context

Reply:
Sorry, I'm out of my depth. I just thought this was a simple fix that I could help with. I probably will end up creating more work than it's worth.